.gitignore

```diff
@@ -9,7 +9,6 @@ node_modules
 .*
 !.travis.yml
 !.editorconfig
-!.eslintrc.json
```

Comment:
If we will remove this line, then any changes to `.eslintrc.json` will not be tracked by git - are we sure that it should not be tracked?